README.md

```diff
@@ -18,7 +18,7 @@ Status: STABLE. The format will only change if bugs are found. Please report any
 
 Binaries can be downloaded from [Releases](https://github.com/klauspost/asmfmt/releases). Unpack the file into your executable path.
 
-To install the standalone formatter from source using Go framework: `go get -u github.com/klauspost/asmfmt/cmd/asmfmt`.
+To install the standalone formatter from source using Go framework: `go install github.com/klauspost/asmfmt/cmd/asmfmt@v1.3.1`.
```

Comment:
Wouldn't this work?
```suggestion
To install the standalone formatter from source using Go framework: `go install github.com/klauspost/asmfmt/cmd/asmfmt@latest`.
```

Reply:
I'm not used to pull requests. Should I do an extra step to commit your change or is it automatically commited if I resolve the conversation?

Reply:
@fredtux You have to commit it through the UI.